src/Apps/Collect2/Routes/Collect/index.tsx

```diff
@@ -1,153 +1,136 @@
-
-    // client renders will get COLLECTION_HUBS from sd; server renders
-    //   will get it from the SystemContext.
-    const showCollectionHubs =
-      sd.COLLECTION_HUBS === "experiment" ||
-      this.props.COLLECTION_HUBS === "experiment"
-
-    return (
-      <AppContainer>
-        <FrameWithRecentlyViewed>
-          <Title>{title}</Title>
-          <Meta property="og:url" content={`${sd.APP_URL}/collect`} />
-          <Meta
-            property="og:image"
-            content={`${sd.APP_URL}/images/og_image.jpg`}
-          />
-          <Meta name="description" content={description} />
-          <Meta property="og:description" content={description} />
-          <Meta property="twitter:description" content={description} />
-          <Link rel="canonical" href={canonicalHref} />
-          <BreadCrumbList
-            items={[
-              { path: "/collect", name: "Collect" },
-              medium && {
-                path: `/collect/${medium}`,
-                name: breadcrumbTitle,
-              },
-            ].filter(Boolean)}
-          />
-          <SeoProductsForArtworks artworks={filter_artworks} />
+})((props: CollectAppProps) => {
+  const { params, viewer, location } = props
+  const medium = params && params.medium
+  const { description, breadcrumbTitle, title } = getMetadataForMedium(medium)
+  const { trackEvent } = useTracking()
+
+  const canonicalHref = medium
+    ? `${sd.APP_URL}/collect/${medium}`
+    : `${sd.APP_URL}/collect`
+
+  // Client renders will get COLLECTION_HUBS from sd; server renders
+  // will get it from the SystemContext.
+  const showCollectionHubs =
+    sd.COLLECTION_HUBS === "experiment" ||
+    props.COLLECTION_HUBS === "experiment"
+
+  return (
+    <AppContainer>
+      <FrameWithRecentlyViewed>
+        <Title>{title}</Title>
+        <Meta property="og:url" content={`${sd.APP_URL}/collect`} />
+        <Meta
+          property="og:image"
+          content={`${sd.APP_URL}/images/og_image.jpg`}
+        />
+        <Meta name="description" content={description} />
+        <Meta property="og:description" content={description} />
+        <Meta property="twitter:description" content={description} />
+        <Link rel="canonical" href={canonicalHref} />
+
+        <BreadCrumbList
+          items={[
+            { path: "/collect", name: "Collect" },
+            medium && {
+              path: `/collect/${medium}`,
+              name: breadcrumbTitle,
+            },
+          ].filter(Boolean)}
+        />
+
+        <SeoProductsForArtworks artworks={props.filterArtworks} />
 
-          <Serif size="8" mt={3} element="h1">
-            Collect art and design online
+        <Box mt={3}>
+          <Serif size="8">
+            <h1>Collect art and design online</h1>
           </Serif>
-          <Separator mt={2} mb={[2, 2, 2, 4]} />
 
           {showCollectionHubs && (
             <>
+              <Separator mt={2} mb={[2, 2, 2, 4]} />
```

Comment:
Are you sure we don't want to render this when the hubs aren't showing? I'd think we still want a separator between the `h1` and grid.

Reply:
Ah, nice  catch 👍